### PR TITLE
Test for #4713

### PR DESCRIPTION
**Pull request by vtols**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/series/SeriesQueryAddMetaTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        return result;
+    }
+
+    @Issue("4713")
+    @Test(
+            description = "Check that meta is included for all types of data",
+            dataProvider = "seriesQueryTypeProvider"
+    )
+    public void testSeriesResponseMetaIncluded(SeriesQueryType type) throws JSONException {
+        long currentTime = System.currentTimeMillis();
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/series/SeriesQueryAddMetaTest.java]: Will the tests remain reproducible? It seems to me that forecasts can be made also for the series in past.

Comment (rmakulov) [reply src/test/java/com/axibase/tsd/api/method/series/SeriesQueryAddMetaTest.java]: How do you build forecasts - do we need to update api_test image?

Comment (vtols) [reply src/test/java/com/axibase/tsd/api/method/series/SeriesQueryAddMetaTest.java]: There is default forecast, I think we don't need to create a new one specially for this test

Comment (vtols) [reply src/test/java/com/axibase/tsd/api/method/series/SeriesQueryAddMetaTest.java]: But if we want to create forecasts in the past, then possibly we need to create a new forecasting rule

**Pull request merged**